ntbs-service/Helpers/RouteHelper.cs

```diff
@@ -33,5 +34,8 @@ public static class NotificationSubPaths
         public static string LinkedNotifications => "LinkedNotifications";
         public static string Denotify => "Denotify";
         public static string Delete => "Delete";
+
+        public static string EditManualTestResult(int? TestResultId) => $"Edit/ManualTestResult/{TestResultId}";
+        public static string AddManualTestResult => $"Edit/ManualTestResult";

```

Comment:
It looks like overview page called TestResults, while the individual edit pages refers to ManualTestResult - Could we use the same name for both?

Reply:
The two are different: ManualTestResults are just one of the things that will end up on the TestResults page. Does that make it clearer?

Reply:
Ah ok, thanks for clarifying!